nextflow.config

```diff
@@ -34,6 +34,13 @@ params {
   save_reference = null // Built Indexes not saved
   sequencing_center = null // No sequencing center to be written in BAM header in MapReads process
   sentieon = null // Not using Sentieon by default
+  trim_fastq = null // Not trimming by default
+  clip_r1 = null // Integer, Instructs Trim Galore to remove bp from the 5' end of read1 (or single-end reads)
+  clip_r2 = null // Integer, Instructs Trim Galore to remove bp from the 5' end of read 2 (paired-end reads only)
+  three_prime_clip_r1 = null // Integer, Instructs Trim Galore to remove bp from the 3' end of read 1 AFTER adapter/quality trimming has been performed
+  three_prime_clip_r2 = null // Integer, Instructs Trim Galore to remove bp from the 3' end of read 2 AFTER adapter/quality trimming has been performed
+  trim_nextseq = null // Integer, Instructs Trim Galore to apply the --nextseq=X option, to trim based on quality after removing poly-G tails
+  save_trimmed = null // Option to save trimmed FastQ file intermediates
```

Comment:
```suggestion
  save_trimmed = false // Option to save trimmed FastQ file intermediates
```

Reply:
@MaxUlysse Is there any benefit to using `null` to initialise variables in `nextflow.config`? Probably makes more sense to initialise them using the same type of variable e.g. `0` for integers, `false` for boolean etc.